integration-test/data/configs/click-to-load-youtube.json

```diff
@@ -36,14 +36,14 @@
                     "iframe[src*='://www.youtube-nocookie.com/embed']"
                 ],
                 "replaceSettings": {
-                    "type": "youtube-video",
+                    "type": "youtube-video-ctl",
```

Comment:
I'll probably change this flag if we release only the partial changes first.

Reply:
Yea, good idea. If you leave the name alone for now, we can land the bulk of the work without having to mess with the surrogate script in the `tracker-surrogates` repo. We can change it before releasing the feature.